src/analytics.ts

```diff
@@ -353,3 +353,17 @@ export const getCustomerHash = (): Promise<{hash: string}> =>
     postMessageToNativeApp({
         type: 'GET_CUSTOMER_HASH',
     });
+
+export const setTrackingProperty = (
+    system: string,
+    name: string,
+    value?: string,
+): Promise<void> =>
+    postMessageToNativeApp({
+        type: 'SET_TRACKING_PROPERTY',
+        payload: {
+            system: system,
+            name: name,
+            value: value,
+        },
+    });
```

Comment:
I'd catch the exceptions here
```suggestion
    }).catch(() => {});
```